src/main/resources/jira-template.ftl

```diff
@@ -1,16 +1,25 @@
-<#assign rowTotalDuration = 0?number>
 <#if getDescription()?has_content>${getDescription()}</#if>
+
 <#if getNarrativeType() == "NARRATIVE_AND_TIME_ROW_ACTIVITY_DESCRIPTIONS">
  <#list getTimeRows() as timeRow>
-  <#assign rowTotalDuration += timeRow.getDurationSecs()>
   |${timeRow.getActivity()}|${timeRow.getDescription()!"N/A"}|
  </#list>
 </#if>
-<#if getDescription()?has_content || getNarrativeType() == "NARRATIVE_AND_TIME_ROW_ACTIVITY_DESCRIPTIONS">
- ${'\n'}
-Total worked time: ${rowTotalDuration?string.@duration}
+
+Total worked time: ${getTotalDuration(getTimeRows())?string.@duration}
 Total chargeable time: ${getTotalDurationSecs()?string.@duration}
 Experience factor: ${getUser().getExperienceWeightingPercent()}%
+
+<#if getDurationSplitStrategy() == "DIVIDE_BETWEEN_TAGS" && (getTags()?size > 1)>
```

Comment:
We can remove this comment. `workedTime` is already split (or not), and we are not displaying per row durations.